addons/onboarding/models/onboarding_onboarding.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class Onboarding(models.Model):
+    _name = 'onboarding.onboarding'
+    _description = 'Onboarding'
+
+    name = fields.Char('Name of the onboarding', translate=True)
+    onboarding_alias = fields.Char('One word alias', translate=False)
+    step_ids = fields.One2many('onboarding.onboarding.step', 'onboarding_id', 'Onboarding steps')
+
+    is_per_company = fields.Boolean('Should be done per company?', default=True)
+
+    panel_background_color = fields.Selection(
+        [('orange', 'Orange'), ('blue', 'Blue'), ('violet', 'Violet'), ('none', 'None')],
+        string="Panel's Background color", default='orange',
+        help="Color gradient added to the panel's background.")
+    panel_background_image = fields.Binary("Panel's background image")
+    panel_close_action_name = fields.Char(
+        'Closing action', help='Name of the onboarding model action to execute when closing the panel.')
+
+    current_progress_id = fields.One2many(
```

Comment:
Seems it is a Many2one to onboarding.progress, no need of inverse ``onboarding_id``, just a computed many2one :)

Reply:
company_progress_id ?